### PR TITLE
Add reference to tutorial page in cli

### PR DESCRIPTION
**Pull request by tsukudamayo**
## Motivation

- a subtask of https://github.com/optuna/optuna/issues/2940

## Description of the changes

- add a link to cli tutorial.

### CODE REVIEW

File: docs/source/reference/cli.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ optuna.cli
 
 The :mod:`~optuna.cli` module implements Optuna's command-line functionality using the `cliff framework <https://docs.openstack.org/cliff/latest/index.html>`_.
 
+.. seealso::
+   Examples are provided in the :ref:`cli` tutorial.
```
Comment (toshihikoyanase) [inline docs/source/reference/cli.rst:9]: I guess the tutorial page does not show an example but also shows use cases. How about rephrasing it, for example, as follows?
```suggestion
    The :ref:`cli` tutorial provides use-cases with examples.
```

**Review by toshihikoyanase — COMMENTED** [review]
Thank you for your PR! I added a suggestion about the description.

**Review by nzw0301 — APPROVED** [review]
LGTM. Thanks

**Review by toshihikoyanase — APPROVED** [review]
LGTM! Thank you for your contribution!

**Pull request merged**